src/main/java/com/geespring/marketapi/controller/ProductController.java

```diff
@@ -0,0 +1,73 @@
+package com.geespring.marketapi.controller;
+
+import com.geespring.marketapi.model.Product;
+import com.geespring.marketapi.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "products")
```

Comment:
Can we make this singular `"product"` to match the database resource and be more consistent with other controllers?